src/pythonic/index.md

```diff
@@ -116,6 +116,23 @@ for conveying what you're doing.
 > Also, make sure you document your exported functions. From experience, working with 
 > libraries with non-existent documentation is not fun. Please don't add to the problem!
 
+The contents of [Cargo.toml](./pythonic/primes/Cargo.toml) are below:
+```
+[package]
+name = "primes"
+version = "0.1.0"
+authors = ["Michael Bryan <michaelfbryan@gmail.com>"]
+
+[dependencies]
+primal = "*"
+libc = "*"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+```
+
+Adding `cdylib` to the `lib` section compiles the crate as a system library
+usable by other languages.
```

Comment:
[This](https://doc.rust-lang.org/reference/linkage.html) Rust reference page on linking gave a good overview on crate-types and their recommended usage.  So linking to it would be useful for more info.
So, it could read something like
"
```Adding `cdylib` to the `lib` section compiles the crate as a system library
usable by other languages.``` 
More info on crate-types can be found [here](https://doc.rust-lang.org/reference/linkage.html).  For a refresher on static vs dynamic linking see [here](http://stackoverflow.com/questions/2649334/difference-between-static-and-shared-libraries)
"